app/src/passwordStrength.py

```diff
@@ -0,0 +1,38 @@
+'''
+Password Validity
+1. Minimum 8 characters.
+2. The alphabet must be between [a-z]
+3. At least one alphabet should be of Upper Case [A-Z]
+4. At least 1 number or digit between [0-9].
+5. At least 1 character from [ _ or @ or $ ].
+'''
+
+
+from password_strength import PasswordPolicy
+
+
+# Define password policy
+policy = PasswordPolicy.from_names(
+    length=8,  # Minimum length
+    uppercase=1,  # At least 1 uppercase letter
+    numbers=1,  # At least 1 digit
+    special=1,  # At least 1 special character
+)
+
+def check_password_strength(password):
+    result = policy.test(password)
+    if not result:
+        return None #password is strong enough
+    return result #return a list of suggestions
+
```

Comment:
The password strength checker function is well-implemented and it ensures that we accept strong passwords with the usage of password policy